src/index.ts

```diff
@@ -6,7 +6,7 @@ import fetchToml from './network'
 import {
   normalizeManifest,
   ManifestParsed,
-  ManifestRPC,
+  StreamManifest,
```

Comment:
Do we want to add a `ManifestRPC` type that matches the response of the `manifest` RPC?

Reply:
Yes will do that and request re-review

Reply:
Actually I think I'll do that in a separate PR. It looks like ManifestParsed is really similar to what an RPCManifest would look like so I'll just replace that.  Here's what the manifest from the RPC call returns: 
`{
  Sequence: 1,
  PublicKey: 'EDA4074FD039407BD2464F14C378440D5B02CA8FBA661B286D1C82A3D59E8E6EC0',
  SigningPubKey: '0323A8C668E7F99A7FF13714D8FFAA24010A466820DB3896F1E3FF06C84ECCCE5C',
  Signature: '30450221009C5B5BF6DEA0C97CD758A0488FB93F707816777DC1561CF6C7745DB2A12261680220664C7EE49EBC37C5689F3050A5666D3D04BD1362E97673EE68F19A78DCC13147',
  MasterSignature: 'D982C08824328A9AEE2AAA237F574CC2C55A5AEA4905AC31D1DCAAFC59B95314F98076712B001647A97BF462B247CF0B7C89B1CDA4E144B99C2432A97759E504'
}`

Reply:
If it matches the `manifestParsed` type then `normalizeManifest()` should still work.

Reply:
Else we can add it in another PR

Reply:
It matches the names names of the properties, but the values are all hex string that need to be converted which involves changing some of the functions so I'll do that separately. I was able to use this branch in the validator-history-service to get the manifests ingest going.